src/Control/Comonad/Trans/Env.hs

```diff
@@ -59,15 +59,19 @@ import Control.Comonad
 import Control.Comonad.Hoist.Class
 import Control.Comonad.Trans.Class
 import Data.Data
+import Data.Functor.Classes
 import Data.Functor.Identity
 import GHC.Generics
+import GHC.Read (expectP)
+import Text.Read (Read (..), parens)
+import Text.Read.Lex (Lexeme(..))
 
 -- $setup
 -- >>> import Control.Comonad
 
 type Env e = EnvT e Identity
 data EnvT e w a = EnvT e (w a)
-  deriving (Generic, Generic1, Data)
+  deriving (Eq, Ord, Read, Show, Generic, Generic1, Data)
```

Comment:
Deleting these instances seems odd. Is this a mistake introducing during merging?

Reply:
This line only adds instances – the previous ones are still there.